test/Tester/RelationalUtilities/PostgreSqlStorageForTesting.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
```

Comment:
As you can see, there's a bit cruft here and with regards to `RelationalStorageForTesting` and `TestEnvironmentInvariant`. You seem to have cleared them, but FYI, we ought to refactor at some point. The latter is used only in storage provider tests, by the way and the `PostgreSQL` native `UPSERT` in the newest might be useful there (the SQL Server and MySQL one split the more expensive insert from update, since updates are more frequent).